README.md

```diff
@@ -54,6 +54,13 @@ LoggingSystem.bootstrap(StreamLogHandler.standardError)
 
 For further information, please check the [API documentation][api-docs].
 
+#### Selecting a logging backend implementation (applications only)
```

Comment:
Following the style of https://github.com/apple/swift-metrics#selecting-a-metrics-backend-implementation-applications-only and right after the default logger behavior is described; seems like a good spot for this